src/postgres/getClientConfig.ts

```diff
@@ -48,51 +50,6 @@ export async function getClientConfig(treeItem: PostgresServerTreeItem, database
         await client.connect();
         return clientConfig;
     } finally {
-        await client.end();
+        client.end;
     }
 }
-
-export const BaltimoreCyberTrustRoot: string = `-----BEGIN CERTIFICATE-----
```

Comment:
I liked it better when these were here. They're only used in this file and they're kind of long/ugly constants, so I don't think I wanna clutter the `constants.ts` file